File-Formats/PDF/Working-with-forms.md

```diff
@@ -9,6 +9,9 @@ documentation: UG
 
 An interactive form, sometimes referred to as an AcroForm is a collection of fields for gathering information. A PDF document can contain any number of fields appearing on any combination of pages, all of that make a single, globally interactive form spanning the entire document.
 
+To quickly get started with creating, filling, flattening PDF form fields in .NET using the Syncfusion PDF library, refer to this video tutorial:
```

Comment:
To quickly get started with creating, filling, and flattening PDF form fields in .NET using the Syncfusion PDF library, refer to this video tutorial: